src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/DataGuidanceFileWriter.cs

```diff
@@ -175,6 +175,43 @@ private static async Task WriteDataFiles(IList<MetaGuidanceSubjectViewModel> sub
                         );
                     }
 
+                    var footnotes = subject.Footnotes
+                        .Where(footnote => !footnote.Label.IsNullOrWhitespace())
+                        .ToList();
+
+                    if (footnotes.Any())
+                    {
+                        await file.WriteLineAsync();
+                        await file.WriteLineAsync("Footnotes:");
+                        await file.WriteLineAsync();
+
+                        await footnotes
+                            .ForEachAsync(
+                                async (footnote, footnoteIndex) =>
+                                {
+                                    await file.WriteAsync($"{footnoteIndex + 1}. ");
+
+                                    var indexWidth = (footnoteIndex + 1).ToString().Length;
+                                    var indent = string.Empty.PadRight(indexWidth + 2);
```

Comment:
Same comment as per the `baseIndent` comment added previously here - seeing as you've already constructed the line item header value `$"{footnoteIndex + 1}. "` above, can't you use that same string to get the indent width rather than calculating it again?

Reply:
Good suggestion, I've changed it to this!